### ISSUE TITLE
In situ pulmonary artery thrombosis

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6520]
**Preferred term label:**  _In situ_ pulmonary artery thrombosis

**Synonyms**

**Definition (free text, please give PubMed ID)**
Localised thrombosis in pulmonary arteries frequently found in patients with idiopathic and hereditary pulmonary arterial hypertension and pulmonary arterial hypertension associated with congenital heart disease.
PMID: 15194174
PMID: 15194175
PMID: 16899857
PMID: 6148159

**Parent term (use hpo.jax.org/app)**
Abnormal pulmonary artery morphology HP:0030966

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**

Idiopathic pulmonary arterial hypertension ORPHA:275766
Hereditary pulmonary arterial hypertension ORPHA:275777
Pulmonary arterial hypertension associated with congenital heart disease

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
@eswietlik  Is it necesssary to specify "is situ"? Is there another location for pulmonary artery thrombosis

**Comment by eswietlik** [comment]
> @eswietlik Is it necesssary to specify "is situ"? Is there another location for pulmonary artery thrombosis

"In situ" refers to the place from which the thrombosis originated, meaning originated in the place where we see it rather than being dislodged from one place (deep veins) to another (pulmonary artery)

**Comment by pnrobinson** [comment]
I am looking for a PubMed article in which the clinical manifestations are described. I have found a number of case reports but am not sure if there is a better review article we could use to derive our definition.

**Comment by eswietlik** [comment]
PMID: 15194174
PMID: 15194175
PMID: 16899857
PMID: 6148159

**Comment by pnrobinson** [comment]
So it seems that this is an inference in patients without deep vein thrombosis but there is no phenotypic distinction

A high prevalence of isolated
PE may suggest localized thrombus formation in the
pulmonary arteries instead of embolization from per-
ipheral clots. In situ thrombosis is not differentiated
from emboli on CTA. However, despite evidence for
multiple possible origins for clots within the pulmonary
arteries, PE is the default diagnosis for filling defects
detected by CTA in clinical practice. 

https://journal.chestnet.org/article/S0012-3692(20)30467-0/fulltext

**Comment by eswietlik** [comment]
Discussed with two radiologists both agree that they can tell apart PE form in situ thrombosis, this is in accord with my clinical practice.

**Comment by pnrobinson** [comment]
added term as suggested, thanks
